lua/entities/gmod_wire_expression2/core/custom/applyforce.lua

```diff
@@ -0,0 +1,194 @@
+local ExtName = "applyforce"
+
+E2Lib.RegisterExtension(ExtName, true, "Allows E2 chips to applyforce", "Allows E2 chips to applyforce"
+
+function ApplyForceValidAction()
```

Comment:
This has to be defined before you register the extension, otherwise if it's disabled this function will never be created